libevm/VMFactory.cpp

```diff
@@ -82,6 +82,38 @@ void validate(boost::any& v, const std::vector<std::string>& values, VMKind* /*
     throw po::validation_error(po::validation_error::invalid_option_value);
 }
 
+namespace
+{
+/// The name of the program option --evmc. The boost will trim the tailing
+/// space and we can reuse this variable in exception message.
+const char c_evmcPrefix[] = "evmc ";
+
+/// The list of EVM-C options stored as pairs of (name, value).
+std::vector<std::pair<std::string, std::string>> s_evmcOptions;
+
+/// The additional parser for EVM-C options. The options should look like
+/// `--evmc name=value` or `--evmc=name=value`. The boost pass the strings
+/// of `name=value` here. This function splits the name and value or reports
+/// the syntax error if the `=` character is missing.
```

Comment:
How to pass multiple envs? `--evmc debug=true --evmc tracing=true` ?

Reply:
Yes.